### DISCUSSION TITLE
Does it not support sqlite json functions?(Json1 - JSON Integration)

### DISCUSSION THREAD

**Discussion by Keshavsaini137** [tekartik/sqflite#792] [Q&A]
Does it not support sqlite json functions?(Json1 - JSON Integration)
if not is there any way we can get it working it?

Using Sqlite triggers in that i'm using json functions.
But getting error "DatabaseException(no such function: json_group_object (code 1 SQLITE_ERROR):"

**Comment by alextekartik** [comment]
See information here #70

Maybe sqlite3 supports it, in this case you could use sqflite_common_ffi and bundle a custom sqlite build

**Reply by Keshavsaini137** [reply]
I'm new to this How do you bundle a custom sqlite build ?

**Reply by alextekartik** [reply]
You can start from here to use sqflite_common_ffi first: https://github.com/tekartik/sqflite/blob/master/sqflite_common_ffi/doc/using_ffi_instead_of_sqflite.md

sqflite3 package (not my package) is used so you can check its documentation. It has some ready to use flutter libs. Maybe they support json1 already
